uitools/import/Esri/ArcGISRuntime/Toolkit/Callout.qml

```diff
@@ -54,11 +54,13 @@ import QtQuick.Shapes
      \newcode
             Callout {
               calloutData: myCalloutData
-              titleTextColor: "#000000"
-              backgroundColor: "#ffffff"
-              borderColor: "#000000"
-              borderWidth: 2
-              cornerRadius: 5
+              palette.windowText: "#000000"
+              background: Rectangle {
```

Comment:
❓ I was just wondering whether a `pane` would make more sense to be used here than `rectangle`?

Reply:
It would give more functionalities, but I am just overriding the template Callout in calcite which has a Rectangle. I don't think we need anything complicated for the background, and from the Pane docs I cannot find the radius property.